Gruntfile.js

```diff
@@ -43,12 +43,13 @@ module.exports = function(grunt) {
         logConcurrentOutput: true
       }
     },
-    env : {
-      options : {},
+    env: {
+      options: {},
       // environment variables - see https://github.com/jsoverson/grunt-env for more information
       local: {
         src: "gamma.env",
         FH_USE_LOCAL_DB: true,
+        WFM_AUTH_GUID: "iidn3tvprs62asdebat5m3eg",
```

Comment:
Can you add a comment to say what this is for (used for local development when running the authentication service locally)